packages/@aws-cdk/aws-cloud9/test/cloud9.environment.test.ts

```diff
@@ -79,9 +80,49 @@ test('throw error when subnetSelection not specified and the provided VPC has no
 test('can use CodeCommit repositories', () => {
   // WHEN
   const repo = codecommit.Repository.fromRepositoryName(stack, 'Repo', 'foo');
+  const user = new iam.User(stack, 'User');
+  new cloud9.Ec2Environment(stack, 'C9Env', {
```

Comment:
You updated an existing test here. Please make a new

```ts
test('my new test', () => {
  // ...
});
```

block for each test you're adding.